### PR TITLE
added google charts library to create an interactive map on tea.html …

### PR DESCRIPTION
**Pull request by trinitydonohugh**
…page

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -94,6 +94,7 @@ function deleteComment(comment) {
   fetch('/delete-data', {method: 'POST', body: params});
 }
 
+//adds map to tea.html page
```
Comment (isaacwdavis) [inline portfolio/src/main/webapp/script.js:97]: nit: add space and capitalize first letter. Also add period. Comments should be sentences.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -135,3 +136,25 @@ function attachMessage(marker, place) {
     infowindow.open(marker.get("map"), marker);
   });
 }
+
+//draws a chart using google charts library
```
Comment (isaacwdavis) [inline portfolio/src/main/webapp/script.js:140]: ditto

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -135,3 +136,25 @@ function attachMessage(marker, place) {
...
+    ['Germany', 1700],
+    ['South Korea', 1100],
+    ['Canada', 1100],
+    ['Italy', 1000],
+    ['United Kingdom', 1000],
+    ['Japan', 1000],
+    ['France', 800]
+  ]);
+
+  var options = {};
```
Comment (pengxianzhe) [inline portfolio/src/main/webapp/script.js:155]: If the option is empty, you don't need to declare a variable for it.

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (trinitydonohugh) [reply portfolio/src/main/webapp/script.js:97]: ok thank you! I'll make these changes now and push them before merging into master.

**Pull request merged**